src/mesido/asset_sizing_mixin.py

```diff
@@ -782,8 +790,25 @@ def _make_max_size_var(name, lb, ub, nominal):
             lb = 0.0 if np.isinf(bounds[f"{asset_name}.Heat_demand"][1]) else ub
             _make_max_size_var(name=asset_name, lb=lb, ub=ub, nominal=ub / 2.0)
 
-        for asset_name in self.energy_system_components.get("ates", []):
-            ub = bounds[f"{asset_name}.Heat_ates"][1]
+        for asset_name in self.energy_system_components.get("cold_demand", []):
+            ub = (
+                bounds[f"{asset_name}.Cold_demand"][1]
+                if not np.isinf(bounds[f"{asset_name}.Cold_demand"][1])
+                else bounds[f"{asset_name}.HeatIn.Heat"][1]
+            )
+            # Note that we only enforce the upper bound in state enabled if it was explicitly
+            # specified for the demand
+            lb = 0.0 if np.isinf(bounds[f"{asset_name}.Cold_demand"][1]) else ub
+            _make_max_size_var(name=asset_name, lb=lb, ub=ub, nominal=ub / 2.0)
+
+        for asset_name in [
+            *self.energy_system_components.get("ates", []),
+            *self.energy_system_components.get("low_temperature_ates", []),
+        ]:
+            try:
+                ub = bounds[f"{asset_name}.Heat_ates"][1]
+            except KeyError:
+                ub = bounds[f"{asset_name}.Heat_low_temperature_ates"][1]
```

Comment:
it's ok, but this is equivalent, as the Heat_ates bound can only exist if it is an ates.

Reply:
I know that it can only exist if it is an ates, but I prefer it to be explicit and not use a try except when not necessary. 
Also for consistency it is nicer as we normally check if it is a specific asset type

Reply:
done